server/helper.ts

```diff
@@ -240,6 +241,25 @@ function externalApiV2() {
       return;
     }
 
+    //Get Visitors Information For Prom Metrics
+    const ip: string = req.headers['x-forwarded-for'] as string || req.socket.remoteAddress as string;
```

Comment:
`req.headers['x-forwarded-for'] as string` should be `req.headers['x-forwarded-for'] as string | undefined`

Please don't use type assertions like `as` unless you can verify the type.